package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "womenandweblit.mozilla.org",
+  "version": "1.0.0",
+  "description": "Women and Web Literacy",
+  "scripts": {
+    "autoprefix": "node scripts/autoprefix.js",
+    "build:folders": "shx mkdir -p dest/css",
```

Comment:
Suggestion: Can we add line breaks between related tasks? I remember seeing it in another project and it looked purrrty. (Feel free to ignore though)